src/Sylius/Bundle/FixturesBundle/DataFixtures/ORM/LoadProductsData.php

```diff
@@ -292,8 +299,9 @@ protected function addMasterVariant(ProductInterface $product, $sku = null)
      * @param ProductInterface $product
      * @param string           $name
      * @param string           $value
+     * @param array            $translations
```

Comment:
You could fix parameters aligning (they're vertically aligned which is not a standard for *Sylius* anymore ;))